### PR TITLE
SUPINT-2380: nuxeo admin console web scripts works

### PR DESCRIPTION
**Pull request by rakeshkumar1019**

### CODE REVIEW

File: nuxeo-admin-console-package/src/main/resources/package.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- See documentation at http://doc.nuxeo.com/x/HgIz -->
+<package type="addon" name="nuxeo-admin-console-ui-package" version="1.0.0">
+  <title>Nuxeo Admin Console UI package</title>
+  <description>
+    <p>Nuxeo Admin console UI is a standard base web application for Nuxeo admin users.</p>
+  </description>
+  <home-page>http://localhost:8080/nuxeo/admin-console</home-page>
```
Comment (madhurkulshrestha-hyland) [inline nuxeo-admin-console-package/src/main/resources/package.xml:7]: Either remove or edit this URL

File: nuxeo-admin-console-web/ang-src/src/main/resources/OSGI-INF/admin-console-auth-contrib.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<component name="org.nuxeo.admin.console.authentication.angular.admin-console">
+
+  <extension point="startURL" target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService">
+    <startURLPattern>
+      <patterns>
+        <pattern>app/</pattern>
```
Comment (madhurkulshrestha-hyland) [inline nuxeo-admin-console-web/ang-src/src/main/resources/OSGI-INF/admin-console-auth-contrib.xml:6]: should the pattern be admin-console/ ?

File: nuxeo-admin-console-web/ang-src/src/main/resources/OSGI-INF/deployment-fragment.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <!-- Unzip the contents of our web application into the server -->
+    <unzip from="${bundle.fileName}" to="/">
+      <include>nuxeo.war/**</include>
+    </unzip>
+  </install>
+
+  <extension target="web#STD-AUTH-FILTER">
+    <filter-mapping>
+      <filter-name>NuxeoAuthenticationFilter</filter-name>
+      <url-pattern>/app/*</url-pattern>
```
Comment (madhurkulshrestha-hyland) [inline nuxeo-admin-console-web/ang-src/src/main/resources/OSGI-INF/deployment-fragment.xml:14]: should the pattern be admin-console/ ?

**Review by madhurkulshrestha-hyland — CHANGES_REQUESTED** [review]

File: nuxeo-admin-console-web/ang-src/src/main/resources/OSGI-INF/admin-console-auth-contrib.xml (hunk 2 above)
Comment (rakeshkumar1019) [reply nuxeo-admin-console-web/ang-src/src/main/resources/OSGI-INF/admin-console-auth-contrib.xml:6]: its a  directory path

File: nuxeo-admin-console-package/src/main/resources/package.xml (hunk 1 above)
Comment (rakeshkumar1019) [reply nuxeo-admin-console-package/src/main/resources/package.xml]: done

File: nuxeo-admin-console-web/ang-src/src/main/resources/OSGI-INF/deployment-fragment.xml (hunk 3 above)
Comment (rakeshkumar1019) [reply nuxeo-admin-console-web/ang-src/src/main/resources/OSGI-INF/deployment-fragment.xml:14]: its a  directory path

**Review by Nishant0928 — APPROVED** [review]

File: nuxeo-admin-console-web/src/main/resources/OSGI-INF/auth-config.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<component name="org.nuxeo.admin.console.web.authentication.contrib">
+  
+  <extension point="startURL" target="org.nuxeo.ecm.platform.ui.web.auth.service.PluggableAuthenticationService">
+    <startURLPattern>
+      <patterns>
+        <pattern>admin-console</pattern>
```
Comment (madhurkulshrestha-hyland) [inline nuxeo-admin-console-web/src/main/resources/OSGI-INF/auth-config.xml:6]: This needs to be modified

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    <groupId>org.nuxeo</groupId>
+    <artifactId>nuxeo-parent</artifactId>
+    <version>2023.0</version>
+  </parent>
+
+  <groupId>org.nuxeo.admin.console</groupId>
+  <artifactId>admin-console</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <name>Admin console</name>
+  <description>admin-console</description>
```
Comment (madhurkulshrestha-hyland) [inline pom.xml:14]: Description should be a full sentence

**Review by madhurkulshrestha-hyland — DISMISSED** [review]

File: README.md [hunk 6]
```diff
@@ -1,5 +1,77 @@
-# nuxeo-admin-console-ui
-Nuxeo Admin console UI is a standard base web application for Nuxeo admin users.
+# Nuxeo Admin console UI
+Nuxeo Admin console UI is a standard base web application for Nuxeo admin useres.
```
[the remark this answers is not in the document]
Comment (rakeshkumar1019) [reply README.md]: done

File: pom.xml [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo</groupId>
+    <artifactId>nuxeo-parent</artifactId>
+    <version>2023.0</version>
+  </parent>
+
+  <groupId>org.nuxeo.admin.console</groupId>
+  <artifactId>admin-console</artifactId>
```
Comment (madhurkulshrestha-hyland) [inline pom.xml:11]: May need modification

File: nuxeo-admin-console-package/pom.xml [hunk 8]
```diff
@@ -0,0 +1,46 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.admin.console</groupId>
+    <artifactId>admin-console</artifactId>
```
Comment (madhurkulshrestha-hyland) [inline nuxeo-admin-console-package/pom.xml:5]: needs modification

**Review by madhurkulshrestha-hyland — CHANGES_REQUESTED** [review]

**Review by madhurkulshrestha-hyland — APPROVED** [review]

**Pull request merged**